model/simulationtests/heatexchanger_desiccant_balancedflow.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'openstudio'
+require 'lib/baseline_model'
+
+model = BaselineModel.new
+
+# make a 2 story, 100m X 50m, 10 zone core/perimeter building
+model.add_geometry({ 'length' => 100,
+                     'width' => 50,
+                     'num_floors' => 1,
+                     'floor_to_floor_height' => 4,
+                     'plenum_height' => 1,
+                     'perimeter_zone_depth' => 3 })
```

Comment:
As far as I can tell, you really want to end up with 2 zones, so you have to 2 AirLoopHVAC to connect to. Your comment is outdated in any case. Bottom line, I suggest using this:

```suggestion
# make a 2 story, 100m X 50m, 2 zone core building
model.add_geometry({ 'length' => 100,
                     'width' => 50,
                     'num_floors' => 2,
                     'floor_to_floor_height' => 4,
                     'plenum_height' => 0,
                     'perimeter_zone_depth' => 0 })
```